thefuck/rules/pip_install.py

```diff
@@ -0,0 +1,15 @@
+from thefuck.utils import for_app
+from thefuck.specific.sudo import sudo_support
+
+
+@sudo_support
+@for_app('pip')
+def match(command):
+    return ('pip install' in command.script and "Permission denied" in command.output)
+
+
+def get_new_command(command):
+    if "--user" not in command.script:  # add --user (attempt 1)
+        return command.script.replace(" install ", " install --user ")
+    else:  # since --user didn't fix things, let's try sudo (attempt 2)
```

Comment:
This else here is unnecessary and should be removed.

Reply:
Quotation has been changed as requested. 
Could you kindly explain why the `else` is unnecessary? The script is supposed to react differently, depending on whether or not the command already included the `--user` flag, in which case sudo might fix the issue.

Reply:
An else after a return is unnecessary and only increases indentation, decreasing readability.

This:
```python
if condition:
    return foo
else:
    return bar
```
Is equivalent to this:
```python
if condition:
    return foo
return bar
```

Reply:
🤦🏻‍♀️ you're absolutely right of course. I thought you asked me to delete the entire `else` block including the 2nd return statement.

Reply:
Changed as requested

Reply:
Oh, pardon me for not being clear about that :relaxed: Glad you changed it! Thanks for your contribution.